### DISCUSSION TITLE
Convert existing field to relation

### DISCUSSION THREAD

**Discussion by apstanisic** [directus/directus#3086] [Feature Requests]
When coming from existing project, or Directus v8, there is no option to change field to relation.
For example, if I have an existing field `category_id`, by default it will be type of UUID or INT, and there is not way to convert it to relation.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
